src/index.js

```diff
@@ -62,26 +62,54 @@ function reorderContent(components, startIndex, endIndex) {
   return result;
 }
 
+function renderToStaticMarkup(
+  content: $ContentBlocks = [],
+  components: $Components = []
+) {
+  return ReactDOMServer.renderToStaticMarkup(
```

Comment:
We need to also extract the CSS classes into a style tag using the JSS plugin like we do when we server-side render- everything  else looks good here so I'm gonna merge and make a separate issue for doing that!